src/Mesh/babylon.abstractMesh.ts

```diff
@@ -1348,8 +1347,11 @@ module BABYLON {
                 subMesh._trianglePlanes = [];
                 var start = subMesh.verticesStart;
                 var end = (subMesh.verticesStart + subMesh.verticesCount);
+
                 for (var i = start; i < end; i++) {
-                    subMesh._lastColliderWorldVertices.push(Vector3.TransformCoordinates(this._positions[i], transformMatrix));
+                    const v = Vector3.FromFloatArray(positionBuffer, 3*i)

```

Comment:
This will overloads the GC with new Vector3 on every frame The goal of the pointsarray is to avoid instancing new Vector3 on each frame